server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+const port = process.env.PUBLIC_PORT || 3000;
+
+app.get('/ping', (req, res) => {
```

Comment:
It would be beneficial to add a catch-all route at the end of your route definitions to handle any requests to undefined routes. This can help improve the user experience by providing a meaningful response instead of a default error page. [medium]